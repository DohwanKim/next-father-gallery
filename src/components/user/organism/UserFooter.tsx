import { Home, Mail, Phone } from 'lucide-react';

import dayjs from '@/utils/dayjs';

const UserFooter = () => {
  return (
    <footer className={'border-t'}>
      <div
        className={
          'flex container text-xs justify-between h-[60px] gap-y-2 items-center flex-wrap py-4 text-foreground/60'
        }
      >
        <div className={'flex gap-x-5 gap-y-2 mr-5 flex-wrap'}>
          <address className={'flex not-italic gap-1 items-center'}>
            <Home className={'h-[1rem] w-[1rem] transition-all'} />
            Sarim Street, ChangWon City, South Korea
          </address>
          <a
            href={'tel:+82 10 3831 5338'}
            className={'flex gap-1 items-center'}
          >
            <Phone className={'h-[1rem] w-[1rem] transition-all'} />
            010-3831-5338
          </a>
          <a
            href={'mailto:kreator2006@naver.com'}
            className={'flex gap-1 items-center'}
          >
            <Mail className={'h-[1rem] w-[1rem] transition-all'} />
            kreator2006@naver.com
          </a>
        </div>
        <div>
          <p>
            @Copyright {dayjs().format('YYYY')}. DongCheol Kim. All rights
            reserved.
          </p>
        </div>
      </div>
    </footer>
  );
};

export default UserFooter;
