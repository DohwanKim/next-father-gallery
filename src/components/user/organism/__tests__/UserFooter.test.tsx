import { render } from '@testing-library/react';

import UserFooter from '@/components/user/organism/UserFooter';

describe('UserFooter', () => {
  it('UserFooter 스냅샷 일치', () => {
    const { container } = render(<UserFooter />);
    expect(container).toMatchSnapshot();
  });
});
