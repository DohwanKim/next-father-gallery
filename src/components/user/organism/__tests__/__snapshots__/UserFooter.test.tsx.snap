// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserFooter UserFooter 스냅샷 일치 1`] = `
<div>
  <footer
    class="border-t"
    data-testid="footer"
  >
    <div
      class="flex container text-xs justify-between min-h-[60px] gap-y-2 items-center flex-wrap py-4 text-foreground/60"
    >
      <div>
        <p>
          @Copyright 
          2024
          . DongCheol Kim. All rights reserved.
        </p>
      </div>
    </div>
  </footer>
</div>
`;
